src/clarity.ts

```diff
@@ -34,17 +35,25 @@ export function start(override: Config = {}): void {
   }
 }
 
-export function pause(): void {
+function restart(): void {
+  start();
+}
+
+export function suspend(): void {
```

Comment:
Do we send specific events when we go through these state transitions (restart/pause/suspend)?

Reply:
Good point. For suspend yes. However, for pause & resume we don't. For now I will use tags to add instrumentation for these actions so we know when these actions happened.

---
In reply to: [368664746](https://github.com/microsoft/clarity-js/pull/166#discussion_r368664746) [](ancestors = 368664746)

Reply:
Could you add a comment here explaining this a bit?

---
In reply to: [368803710](https://github.com/microsoft/clarity-js/pull/166#discussion_r368803710) [](ancestors = 368803710,368664746)

Reply:
Done, thanks.

---
In reply to: [369058546](https://github.com/microsoft/clarity-js/pull/166#discussion_r369058546) [](ancestors = 369058546,368803710,368664746)